.github/workflows/ci.yml

```diff
@@ -0,0 +1,102 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
nit - with bigger pieces of work, we often raise two PRs, with the second targeting the first's branch. Can we just open this to all branches?